### PR TITLE
Feat: Comment service completed

### PR DESCRIPTION
**Pull request by intyouss**
**What did I do?**
- Fixed Comment service bug
- Completed connection to User service
- Added required docker-compose files
- Complete the test and add the test file

**Thank you to everyone who contributed!**

### CODE REVIEW

**Review by mo3et — APPROVED** [review]
LGTM. Thanks!

File: app/comment/service/deploy/docker-compose.yaml
Comment (alilestera) [inline app/comment/service/deploy/docker-compose.yaml:1]: this docker-compose file should not be commited

Comment (intyouss) [reply app/comment/service/deploy/docker-compose.yaml:1]: The docker-compose files for mysql and redis for the comment service are in the comment directory, not the global directory. This is the standalone storage for comment.

Comment (alilestera) [reply app/comment/service/deploy/docker-compose.yaml:1]: > The docker-compose files for mysql and redis for the comment service are in the comment directory, not the global directory. This is the standalone storage for comment.

You mean that every service should have own database and cache component? 

In fact it's a good practice, but with more services developing, the more containers will be created. That will cost a lot of memory, it's great pressure to personal computer.

AND the better practice I think is that use a single docker-compose file to build all the services and its dependent resources. like:
```docker-compose
services:
  user-service:
    ...
  user-service-database:
    ...
```
BUT if the service need many resources, pick it out to a single docker-compose file is ok.

Comment (intyouss) [reply app/comment/service/deploy/docker-compose.yaml:1]: It's not limited to one host and what we're trying to do should be to manage the table structure individually, so try to move to distributed storage and if the IMAGE version is the same, it's just a difference in containers, it won't take up too much of it, it's just a way to give others the understanding that we're doing distributed storage.

The summary of my point is that while we don't have a lot of room to implement distributed services, it's nice to be able to make the point that we're moving in that direction

Comment (alilestera) [reply app/comment/service/deploy/docker-compose.yaml:1]: OK, we can solve this problem if we really face it.

BUT one service manage own database table not mean that must assign a whole database container to it. Different table would not affect each other in normal.

**Review by alilestera — APPROVED** [review]

**Pull request merged**